addon/components/cell.js

```diff
@@ -76,22 +55,24 @@ export default Component.extend(PropTypeMixin, {
   /**
    * Get definition for current cell
    * @param {BunsenCell} cellConfig - current cell
-   * @param {BunsenCell[]} cellDefinitions - list of cell definitions
+   * @param {Object<String, BunsenCell>} cellDefinitions - list of cell definitions
```

Comment:
That is correct. I wasn't really sure how to properly annotate that and really didn't want to just have `Object` as that doesn't tell you much.